test/e2e/hack/e2e.sh

```diff
@@ -41,72 +68,39 @@ unset KUBECONFIG
 popd
 mv $REPO_DIR/hack/deploy/etcd/statefulset.yaml.tmp $REPO_DIR/hack/deploy/etcd/statefulset.yaml
 
-# TODO use helm instead of kustomize way
 namespace=multicluster-controlplane
 echo "Deploy standalone controlplane in namespace $namespace ..."
-cp -r ${REPO_DIR}/hack/deploy/controlplane/* $deploy_dir
-
-kubectl --kubeconfig ${kubeconfig} delete ns $namespace --ignore-not-found
-kubectl --kubeconfig ${kubeconfig} create ns $namespace
-
-# expose apiserver
-sed -i 's/ClusterIP/NodePort/' $deploy_dir/service.yaml
-sed -i '/route\.yaml/d' $deploy_dir/kustomization.yaml
-sed -i "/targetPort.*/a  \ \ \ \ \ \ nodePort: $external_host_port" $deploy_dir/service.yaml
-
-# append etcd certs
-certs_dir=$deploy_dir/certs
-mkdir -p ${certs_dir}
-cp -f ${REPO_DIR}/multicluster_ca/ca.pem ${certs_dir}/etcd-ca.crt
-cp -f ${REPO_DIR}/multicluster_ca/client.pem ${certs_dir}/etcd-client.crt
-cp -f ${REPO_DIR}/multicluster_ca/client-key.pem ${certs_dir}/etcd-client.key
-sed -i "$(sed -n  '/  - ocmconfig.yaml/=' $deploy_dir/kustomization.yaml) a \  - ${certs_dir}/etcd-client.key" $deploy_dir/kustomization.yaml
-sed -i "$(sed -n  '/  - ocmconfig.yaml/=' $deploy_dir/kustomization.yaml) a \  - ${certs_dir}/etcd-client.crt" $deploy_dir/kustomization.yaml
-sed -i "$(sed -n  '/  - ocmconfig.yaml/=' $deploy_dir/kustomization.yaml) a \  - ${certs_dir}/etcd-ca.crt" $deploy_dir/kustomization.yaml
-
-# create multicluster-controlplane configfile
-cat > ${deploy_dir}/ocmconfig.yaml <<EOF
-dataDirectory: /.ocm
-apiserver:
-  externalHostname: 127.0.0.1
-etcd:
-  mode: external
-  prefix: $namespace
-  caFile: /controlplane_config/etcd-ca.crt
-  certFile: /controlplane_config/etcd-client.crt
-  keyFile: /controlplane_config/etcd-client.key
-  servers:
-  - http://etcd-0.etcd.multicluster-controlplane-etcd:2379
-  - http://etcd-1.etcd.multicluster-controlplane-etcd:2379
-  - http://etcd-2.etcd.multicluster-controlplane-etcd:2379
-EOF
-sed -i "s@ocmconfig.yaml@${deploy_dir}/ocmconfig.yaml@g" $deploy_dir/kustomization.yaml
-
-pushd $deploy_dir
-kustomize edit set namespace $namespace
-kustomize edit set image quay.io/open-cluster-management/multicluster-controlplane=${IMAGE_NAME}
-popd
-
-kustomize build $deploy_dir | kubectl --kubeconfig ${kubeconfig} -n $namespace apply -f -
 
-wait_command "kubectl --kubeconfig $kubeconfig -n multicluster-controlplane get secrets multicluster-controlplane-kubeconfig"
+${KUBECTL} --kubeconfig ${kubeconfig} delete ns $namespace --ignore-not-found
+${KUBECTL} --kubeconfig ${kubeconfig} create ns $namespace
+
+# copy etcd ca to helm folder
+helm_dir=${REPO_DIR}/charts/multicluster-controlplane/multicluster_ca
+mkdir -p ${helm_dir}
+cp -f ${REPO_DIR}/multicluster_ca/ca.pem ${helm_dir}/
+cp -f ${REPO_DIR}/multicluster_ca/client.pem ${helm_dir}/
+cp -f ${REPO_DIR}/multicluster_ca/client-key.pem ${helm_dir}/
```

Comment:
and we may create the etc certs in the `$etc_certs` directly

the helm command will be
```
helm ... "caFile":"${etc_certs}"
```